src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -1469,6 +1469,54 @@ bool AletheProofPostprocessCallback::update(Node res,
                                   : std::vector<Node>(),
                               *cdp);
     }
+    // ======== Alpha Equivalence
+    // See proof_rule.h for documentation on the ALPHA_EQUIV rule. This
+    // comment uses variable names as introduced there.
+    //
+    // Let F = (forall ((y1 A1) ... (yn An)) G) and F*sigma = (forall ((z1 A1)
+    // ... (zn An)) G*sigma)
+    //
+    //
+    //  ------------------ refl
+    //  (cl (= G G*sigma))
+    // -------------------- bind, z1 ... zn (= y1 z1) ... (= yn zn)
+    //  (= F F*sigma)
+    //
+    case ProofRule::ALPHA_EQUIV:
+    {
+      std::vector<Node> varEqs;
+      // performance optimization
```

Comment:
This should go into the comment above in my opinion. But its not important to me so if you have a strong preference, please leave it here.

Reply:
Good point. I improved the documentation.